src/directives/sf-field.js

```diff
@@ -110,6 +110,24 @@ angular.module('schemaForm').directive('sfField',
                             return (expression && $interpolate(expression)(locals));
                         };
 
+                        scope.modelValue = function(valueToSet) {
```

Comment:
I think actually we still need to access the scope from the link function to get the right one.

the call to scope.modelValue() is just a shortcut for "sfModelValue(scope, valueToSet)", which is actually a call to sfModelValue(scope, scope.form.key, scope.model, valueToSet)